gce-containers-startup/utils/volumes.go

```diff
@@ -146,5 +152,193 @@ func processHostPathVolume(volume *api.HostPathVolume) (VolumeHostPathAndMode, e
 }
 
 func processGcePersistentDiskVolume(volume *api.GcePersistentDiskVolume) (VolumeHostPathAndMode, error) {
-	return VolumeHostPathAndMode{}, fmt.Errorf("Unimplemented.")
+	if volume.FsType != "" && volume.FsType != ext4FsType {
+		return VolumeHostPathAndMode{}, fmt.Errof("Unsupported filesystem type: %s", volume.FsType)
+	}
+	volume.FsType = ext4FsType
```

Comment:
Why do you do this? Can this not be true at this point? I'm feeling a little uneasy with this assignment here :-)

Reply:
I made it a local variable.